lib/osf-components/addon/components/file-browser/component.ts

```diff
@@ -284,7 +284,7 @@ export default class FileBrowser extends Component {
                 return;
             }
 
-            const otherItem = this.selectedItems.get('firstObject') as File;
+            const otherItem = this.selectedItems.firstObject as unknown as File;
```

Comment:
Well _that's_ an interesting construction.

Reply:
Yes, it is. Basically, it says, "throw away the type information, re-type it as this". Technically, this should be typed as:
```ts
const otherItem = this.selectedItems.firstObject as ComputedProperty<File> & File;
```
I think I'll make a type alias for this and use it instead,